app/components/filters/recognition-status-select.hbs

```diff
@@ -1,10 +1,17 @@
-<AuLabel class="au-u-margin-top" for="filter-municipality">Erkenning</AuLabel>
-<AuCheckboxGroup
-  @alignment="default"
-  @selected={{@selected}}
-  @onChange={{@onChange}}
-  as |Group|
->
-  <Group.Checkbox @value="Erkend">Erkend</Group.Checkbox>
-  <Group.Checkbox @value="Verlopen">Verlopen</Group.Checkbox>
-</AuCheckboxGroup>
\ No newline at end of file
+<AuFieldset class="au-u-margin-top" as |f|>
```

Comment:
The fieldset change wasn't really needed, but it bothered me that we were using a label here :sweat_smile: